### PR TITLE
PMK-550-Doc-Image-Processor_EN

### PR DESCRIPTION
**Pull request by Luize-boyen**
doc improvements and corrections

### CODE REVIEW

File: image-processor/2021-01-14-index.md [hunk 1]
```diff
@@ -15,56 +15,57 @@ permalink_pt-br: "/documentacao/produtos/image-processor/"
 
 The Azion **Image Processor** automates your workflow for processing images.
 
-Optimizing your images reduces their size, but without any perceivable loss of visual quality. This reduces their transfer time and improves the user’s experience. This makes your pages load better and speeds up navigation, without you needing to do practically anything.
+Optimizing your images reduces their size without any noticeable loss of visual quality, which reduces the transfer time of your content and improves the user experience. This increases the fluidity of your pages and the speed of navigation, without you having to do practically anything.
```
Comment (VitorEltz-Azion) [inline image-processor/2021-01-14-index.md:18]:
```suggestion
Processing your images reduces their size without any noticeable loss of visual quality, which reduces the transfer time of your content and improves the user experience. This increases the fluidity of your pages and the speed of navigation, without you having to do practically anything.
```

File: image-processor/2021-01-14-index.md [hunk 2]
```diff
@@ -75,24 +76,29 @@ To configure Azion **Image Processor**, take the following steps and, when neces
...
+
+5. Also select **Optimize Images** in the **Behavior** field;
+
+6. In all other sections, use the appropriate settings for your needs;
 
-From this point on, images in the configured *path* will automatically be optimized. As well as this, **Image Processor** will detect whether the browser is compatible with the WEBP format and, where possible, it will try to automatically convert the format of the image, giving you yet another benefit. BMP images will also be automatically converted to JPEG or WEBP, depending on the browser’s compatibility.
+7. Click the **Save** button to save your rule.
 
-**Real-Time Metrics** gives you a graph showing the *Bandwidth Saving* achieved, enabling you to monitor the savings you make from the optimization.
+From this point on, images from the configured path will be automatically optimized. In addition, the **Image Processor** detects the browser's compatibility with the WEBP format and, when possible, will convert the image format automatically, bringing even more gains for you. BMP images will also be automatically converted to JPEG or WEBP, depending on the browser's compatibility.
```
Comment (VitorEltz-Azion) [inline image-processor/2021-01-14-index.md:89]:
```suggestion
From this point on, images from the configured path will be automatically processed. In addition, the **Image Processor** detects the browser's compatibility with the WEBP format and, when possible, will convert the image format automatically, bringing even more gains for you. BMP images will also be automatically converted to JPEG or WEBP, depending on the browser's compatibility.
```

File: image-processor/2021-01-14-index.md [hunk 3]
```diff
@@ -75,24 +76,29 @@ To configure Azion **Image Processor**, take the following steps and, when neces
...
 
-See below the other features of the product, configured as arguments in the Query String of the image’s URL.
+Follow the *Bandwidth Saving* graph through **Real-Time Metrics** to monitor the traffic savings resulting from the optimization.
```
Comment (VitorEltz-Azion) [inline image-processor/2021-01-14-index.md:91]:
```suggestion
Follow the *Bandwidth Saving* graph through **Real-Time Metrics** to monitor the traffic savings resulting from the processed images.
```

File: image-processor/2021-01-14-index.md [hunk 4]
```diff
@@ -148,52 +152,52 @@ ims=fit-in/WidthxHeight
...
 
 [![bulldog]({{ site.url }}/static/images/docs/image-optimization/bulldog-1280px.jpg?ims=430x20:910x730)]({{ site.url }}/static/images/docs/image-optimization/bulldog-1280px.jpg?ims=430x20:910x730)
 
 ---
 
-## 5. Change image quality {#change-quality}
+## 5. Changing image quality {#change-quality}
 
-The **Image Processor** optimizes your images, reducing the file size and, therefore, its transfer time. The default value for the quality level to be used is 85%, which allows for them to be optimized, without any perceptible loss of visual quality.
+The **Image Processor** optimizes your images, reducing file size and, therefore, the transfer time. The default quality value used is 85%, which provides optimization without any noticeable loss of visual quality.
```
Comment (VitorEltz-Azion) [inline image-processor/2021-01-14-index.md:190]:
```suggestion
The **Image Processor** process your images, reducing file size and, therefore, the transfer time. The default quality value used is 85%, which provides optimization without any noticeable loss of visual quality.
```

**Review by VitorEltz-Azion — CHANGES_REQUESTED** [review]

**Comment by Luize-boyen** [comment]
This one corrected, too! :)

**Pull request merged**